src/executioners/SpectralExecutionerLinearElastic.C

```diff
@@ -0,0 +1,402 @@
+/**********************************************************************/
+/*                     DO NOT MODIFY THIS HEADER                      */
+/* MAGPIE - Mesoscale Atomistic Glue Program for Integrated Execution */
+/*                                                                    */
+/*            Copyright 2017 Battelle Energy Alliance, LLC            */
+/*                        ALL RIGHTS RESERVED                         */
+/**********************************************************************/
+
+#include "SpectralExecutionerLinearElastic.h"
+#include "InputParameters.h"
+#include "RankFourTensor.h"
+
+registerMooseObject("MagpieApp", SpectralExecutionerLinearElastic);
+
+Real
+deltaij(const int i, const int j)
```

Comment:
I'm not a fan of global functions. Could you make this a member, please?

Reply:
btw. this could just be replaced by `(i == j)` wherever the function is called.

Reply:
`bool` to `int` conversion is implicit.

§4.7/4 from the C++ Standard says (Integral Conversion):
> If the source type is bool, the value false is converted to zero and the value true is converted to one.